ref_soft/r_context.c

```diff
@@ -17,14 +17,14 @@ GNU General Public License for more details.
 
 ref_api_t      gEngfuncs;
 ref_globals_t *gpGlobals;
-ref_instance_t RI;
+extern ref_instance_t RI;
```

Comment:
You can remove RI from here, it's defined in r_main.c and declared globally in r_local.h.